src/blocks/Header/Header.scss

```diff
@@ -77,6 +77,8 @@ $backgroundWidth: 1440px;
     &__description {
         margin-top: $indentXS;
         font-weight: normal;
+        font-size: 0.83em;
```

Comment:
we usually use font sizes from defined set https://preview.gravity-ui.com/uikit/?path=/story/typography--common